test/unit/auth_test.js

```diff
@@ -28,12 +28,14 @@ describe('building authorization entries', function () {
     );
 
     let cred = entry.credentials().address();
-    let args = cred.signatureArgs().map((v) => StellarBase.scValToNative(v));
+    let sig = cred.signature();
+
+    expect(sig.switch().name).to.be.equal('scvVec');
+    const args = StellarBase.scValToNative(sig.vec()[0]);
 
     expect(cred.signatureExpirationLedger()).to.equal(123);
-    expect(args.length).to.equal(1);
     expect(
-      StellarBase.StrKey.encodeEd25519PublicKey(args[0]['public_key'])
+      StellarBase.StrKey.encodeEd25519PublicKey(args['public_key'])
```

Comment:
are sig arg keys like `public_key` doc'd elsewhere such as js/xdr/public, or maybe this test is the doc.

Reply:
it's documented as part of the signature docs, here: https://soroban.stellar.org/docs/fundamentals-and-concepts/invoking-contracts-with-transactions#stellar-account-signatures (that should probably be referenced here! added in 70ad49e

Reply:
i did find ref to `public_key` in the js auth conversion - https://github.com/stellar/js-stellar-base/blob/soroban-testnet-xdr/src/auth.js#L176, that key name is a constant which core defines correct?

Reply:
yeah, `public_key` and `signature` are the field names of the struct that Core expects as the signature. that's what I added in the cross-reference!